core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -2890,12 +2891,13 @@ private static class ExtractImplementor extends AbstractRexCallImplementor {
     }
   }
 
-  private static Expression mod(Expression operand, long factor) {
+  private static Expression mod(Expression operand, long factor, boolean floorMod) {
     if (factor == 1L) {
       return operand;
     } else {
-      return Expressions.call(BuiltInMethod.FLOOR_MOD.method,
```

Comment:
convert '?' to 'else if' to be consistent with existing method

also revert your change removing the line-break; short lines are better

Reply:
thanks, done.